app/frontend/src/features/profile/functions/useUpdateHostingPreferences.test.tsx

```diff
@@ -1,5 +1,5 @@
 import { renderHook } from "@testing-library/react-hooks";
-import useUpdateHostingPreferences from "features/profile/useUpdateHostingPreferences";
+import useUpdateHostingPreferences from "features/profile/functions/useUpdateHostingPreferences";
```

Comment:
thoughts on "hooks" folder name instead of "functions"? I think "hooks" is more common, but it should be obvious they are hooks because of the name "use*"